ReadMe.md

```diff
@@ -278,10 +284,14 @@ Conduction | 導電 | どうでん |
 Conductivity | 導電性 | どうでんせい |
 Connector | コネクタ | |
 Current | 電流 | でんりゅう |
+Daisy-chain | 橋渡し | はしわたし |
 Datasheet | データシート | |
+Detection | 検知 | けんち |
 Diode | ダイオード | |
+Display | 表示 | ひょうじ |
```

Comment:
This word is already mentioned under `Computer and Application Usage`

Reply:
removing